### PR TITLE
fix: Revert the naming change for DelegatedVMNIC on master

### PR DESCRIPTION
**Pull request by bohuini**
**Reason for Change**:
Modified the value of DelegatedVMNIC to resolve compatibility issue

**Issue Fixed**:

**Requirements**:

- [ ] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [ ] includes documentation
- [ ] adds unit tests
- [ ] relevant PR labels added

**Notes**:

### CODE REVIEW

**Review by behzad-mir — APPROVED** [review]

**Comment by bohuini** [comment]
@rbtr This requires code owner's approval. Can you please help?

**Review by rbtr — APPROVED** [review]

**Pull request merged**
